workstation-backend/workspace/models.py

```diff
@@ -1,14 +1,16 @@
+from uuid import uuid4
+
 from django.db import models
 from django.contrib.auth.models import User
 
 # Create your models here.
-
-
 class Workspace(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    id = models.UUIDField(primary_key=True, default=uuid4)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Why are we coming back to ForeignKey instead of OneToOneField?

Reply:
Similar to the comment with the component model. Having this as a OneToOneField means: one user can only have one workspace and one workspace can only have one user. Instead what we want is one user can have multiple workspaces and one workspace can only have one user.